docs/ios-sdk.md

```diff
@@ -99,17 +94,29 @@ func play() {
 }
 ```
 
-At this point, the player will fetch the DASH manifest, start rendering by automatically synching to the targetLatency set in playerOptions (or 3 seconds default). For more information about options, properties and methods please see relevant sections below.
+At this point, the player will fetch the DASH manifest, start rendering by automatically synching to the targetLatency set in the remote config or the playerOptions (3 seconds default).
+For more information about options, properties and methods please see relevant sections below.
 For a sample application code utilizing these minimal steps see the LiveryExample sample application section.
 
+## Life Cycle
+
+Your application needs to call the corresponding player methods for each iOS Application LifeCycle callback
+
+```swift
+public func onApplicationDidBecomeActive()
+public func onApplicationWillResignActive()
+public func onApplicationDidEnterBackground()
+public func onApplicationWillEnterForeground()
+```
+
 ### Livery SDK
```

Comment:
Livery SDK is now part of the Life Cycle chapter.
Maybe we can make it a ## and name it something like Advanced?

Reply:
Sorry, that was a mistake when I copied the "Life Cycle" to below the Usage section. I think change it to "## Livery SDK" is enough. And change all the sub-section of it too, of course.

Reply:
ok